src/main/java/com/github/relativobr/supreme/resource/mobtech/SimpleCard.java

```diff
@@ -62,6 +62,10 @@ public class SimpleCard {
       "&bCard Machine Snow", "", "&fChange to Production Snow", "",
       LoreBuilder.machine(MachineTier.BASIC, MachineType.MACHINE), "", "&3Supreme Component");
 
+  public static final SlimefunItemStack CARD_APPLE = new SupremeItemStack("SUPREME_CARD_APPLE", Material.MUSIC_DISC_STRAD,
+      "&bCard Machine Snow", "", "&fChange to Production Apple", "",
```

Comment:
Whoops, saw this too late! 
```suggestion
      "&bCard Machine Apple", "", "&fChange to Production Apple", "",
```

Reply:
wtf